src/main/java/frc/robot/Constants.java

```diff
@@ -94,6 +94,16 @@ public static final class SwerveConstants2020 {
         );
     }
 
+    public static final class TiltedElevatorConstants {
+        public static final int EXTENSION_ID = 19;
+        public static final float EXTENSION_LIMIT = 20; // in inches, TODO
+        
+        public static final double EXT_GR = 14.0 / 64.0;
+        public static final double EXT_CIRCUM = Math.PI * 0.500; // approx circumference of winch
+        public static final double EXTENSION_ROT_TO_M = EXT_GR * EXT_CIRCUM; // rotations to meters
```

Comment:
I typically keep conversion constants like this in the actual subsystem because they rarely change, and may need to be referenced quickly when writing the code, but unsure